app/perfunwind.cpp

```diff
@@ -506,7 +506,7 @@ QDataStream &operator<<(QDataStream &stream, const PerfUnwind::Location &locatio
 
 QDataStream &operator<<(QDataStream &stream, const PerfUnwind::Symbol &symbol)
 {
-    return stream << symbol.name << symbol.binary << symbol.path << symbol.isKernel << symbol.relAddr << symbol.size << symbol.actualPath;
+    return stream << symbol.name << symbol.binary << symbol.path << symbol.isKernel << symbol.isInline << symbol.relAddr << symbol.size << symbol.actualPath;
```

Comment:
please always add the new data at the end such that the format stays somewhat compatible to before